stable/selenium/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- flah00
```

Comment:
@ekirmayer can you check with these users to see if they want to be in the OWNERS file for this chart?

Reply:
/cc @flah00

Reply:
@scottrigby - I only managed to get @flah00. I think we should keep both of them in the OWNERS file.

Reply:
Hi, just saw your message @ekirmayer, I'd like to be added as well.